docs/query-syntax.md

```diff
@@ -45,6 +47,8 @@ WHERE cpu_usage >= 99
 The above example continuously evaluates the filter expression on the new events in the stream `device_utils` to filter out events
 which have `cpu_usage` less than 99. The final events will be streamed to clients.
 
+\
```

Comment:
what's this

Reply:
Hi @jovezhong, I was trying to put in an extra line break, to visually separately the sections better. Referred to this Github section in this [tutorial ](https://www.w3schools.io/file/markdown-line-break/)-- is there a better way to add the line break?